### PR TITLE
Added waterfox 35.0

### PR DESCRIPTION
**Pull request by nickl-**
64 bit optimized build, using intel compiler, of firefox.

### CODE REVIEW

File: Casks/waterfox.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'waterfox' do
+  version '35.0'
+  sha256 '67a6b583a8c3e5a1b7019dd1dc8402e95d0848b674b3a45e0b764234417ac063'
+
+  url 'https://d1th2p59px32bw.cloudfront.net/releases/osx64/installer/Waterfox%2035.0%20Setup.dmg'
+  name 'Waterfox'
+  homepage 'https://www.waterfoxproject.org'
+  license :oss
+  tags :vendor => 'Firefox'
```
Comment (vitorgalvao) [inline Casks/waterfox.rb]: I think you mean “Mozilla”. Either way, are you sure? I see no indication this actually has any relation to them; it appears to have been made by an individual.

File: Casks/waterfox.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'waterfox' do
+  version '35.0'
+  sha256 '67a6b583a8c3e5a1b7019dd1dc8402e95d0848b674b3a45e0b764234417ac063'
+
+  url 'https://d1th2p59px32bw.cloudfront.net/releases/osx64/installer/Waterfox%2035.0%20Setup.dmg'
```
Comment (vitorgalvao) [inline Casks/waterfox.rb]: Please add a `# cloudfront.net is the official download host per the vendor homepage` comment above this line.

File: Casks/waterfox.rb (hunk 1 above)
Comment (nickl-) [inline Casks/waterfox.rb]: That was copied from the firefox cask. 

The source code wasn't changed just recompiled afaik... so for all intents and purpose it remains a firefox browser.

Comment (vitorgalvao) [inline Casks/waterfox.rb]: No, it does not. We can in no way guarantee no changes will be made in the future. "Vendor" means the one that makes it in distributes it. That is not the case here at all, please remove it.

**Comment by ndr-qef** [comment]
Thanks. Merged as c6aa07d, with the following changes:
- interpolated `version` in download `url`;
- added comment confirming the download provider;
- removed vendor information. (As @vitorgalvao mentioned, “vendor” is best intended as whoever controls the distributed binary.)

**Pull request closed without merging**